catalog-info.yaml

```diff
@@ -21,3 +21,32 @@ spec:
     spec:
       repository: elastic/terraform-provider-ec
       pipeline_file: ".buildkite/pipeline.yml"
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-terraform-provider-ec-release
+  description: Buildkite Pipeline for releasing the terraform-provider-ec
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/terraform-provider-ec-release
+
+spec:
+  type: buildkite-pipeline
+  owner: group:control-plane-stateful
```

Comment:
I used `group:control-plane-stateful-applications` in Elasticstack provider. That group contains only our team.

Reply:
I tried to align the Buildkite perms with the GitHub perms (defined [here](https://github.com/elastic/catalog-info/pull/635/files#diff-9f7cd1a723120f0178ff1ce9b1f671ca39c0bb61fc60492541f52630c9b5c0acR28-R39))

@d-carmo who owns the Terraform Provider EC - the `control-plane-stateful` or `control-plane-stateful-applications` team in GitHub?

Reply:
I will fix-up the group ownership in a follow-up PR for both pipelines if adjustments needs to be made.